ban/tests/http/test_housenumber.py

```diff
@@ -35,11 +35,26 @@ def test_get_housenumber_with_unknown_id_is_404(get, url):
 
 
 def test_get_housenumber_with_cia(get, url):
-    housenumber = HouseNumberFactory(number="22", parent__fantoir="1234")
+    housenumber = HouseNumberFactory(number="1", ordinal="A",
+                                     parent__municipality__insee="27638",
+                                     parent__fantoir="276380011")
     resp = get(url('housenumber-resource', id=housenumber.cia,
                    identifier="cia"))
     assert resp.status == falcon.HTTP_200
-    assert resp.json['number'] == "22"
+    assert resp.json['number'] == "1"
+
+
+def test_add_district_to_housenumber():
+    housenumber = HouseNumberFactory(number="1", ordinal="A",
+                                     parent__municipality__insee="27638",
+                                     parent__fantoir="276380011")
+    postcode = PostCodeFactory(municipality=housenumber.parent.municipality)
+    hn2 = HouseNumberFactory(postcode=postcode, number="2", ordinal="")
+    hn1 = HouseNumberFactory(postcode=postcode, number="1", ordinal="")
+    hn2bis = HouseNumberFactory(postcode=postcode, number="2", ordinal="bis")
+    assert postcode.housenumbers[0] == hn1
+    assert postcode.housenumbers[1] == hn2
+    assert postcode.housenumbers[2] == hn2bis
```

Comment:
We also need some tests that does the same checks for the housenumber<=>Group relations (parent, ancestors, mixed).